### DISCUSSION TITLE
can   integrated  in existing vite project?

### DISCUSSION THREAD

**Discussion by kknono** [vuejs/vitepress#1603] [Q&A]
### Is your feature request related to a problem? Please describe.

hello ,
    in my project, need 'docs' function, can use vitepress integrated in the project?
    not necessary to create a new project. 

thx

### Describe the solution you'd like

any guides to do it

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Read the [Contributing Guidelines](https://github.com/vuejs/vitepress/blob/main/.github/contributing.md).
- [X] Check that there isn't already an issue that asks for the same feature to avoid creating a duplicate.

**Comment by lyx-jay** [comment]
Of course, you just need to install vitepress in your project

**Comment by AniaKru95** [comment]
I'm confused, how I can use  vitepress in existing project.
My structure:
>MyProject
>>docs
>>>index.md
>>>getting-started.md

>>index.html
>>package.json

When I run: `npm run docs:dev` - my main project path is started (index.html), how could I display docs/index.md ?
